contrib/go/src/python/pants/contrib/go/tasks/go_fetch.py

```diff
@@ -39,6 +43,8 @@ def register_options(cls, register):
   def cache_target_dirs(self):
     # TODO(John Sirois): See TODO in _transitive_download_remote_libs, re-consider how artifact
     # caching works for fetches.
+    # TODO(Benjy): There is no TODO that I can see in _transitive_download_remote_libs, so
+    # figure out of the TODO above this one is still relevant.
```

Comment:
I think this was a reference to a TODO in `_fetch_pkg`. Fixed this comment and deleted mine.